File: sdk/rust/examples/online_multi_get.rs
use oomrpc::Client;

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let mut client = Client::connect("http://127.0.0.1:50051").await?;

    let keys = vec!["19".into(), "48".into(), "38".into()];
    let features = vec!["account.state".into(), "transaction_stats.transaction_count_7d".into()];
    let response = client.online_multi_get_raw(keys.clone(), features.clone()).await?;
    println!("RESPONSE={:#?}", response);

    let response = client.online_multi_get(keys, features).await?;
    println!("RESPONSE={:#?}", response);

    Ok(())
}

File: sdk/python_v2/examples/online_multi_get.py
#!/usr/bin/env python3
import asyncio
from oomclient import Client

async def main():
    client = await Client.connect("http://localhost:50051")
    features = ["account.state", "transaction_stats.transaction_count_7d"]
    entity_keys = ["48", "1", "5", "9", "27", "6", "41", "72"]
    result = await client.online_multi_get(entity_keys, features)
    print(result)

asyncio.run(main())
